frontend/src/App.tsx

```diff
@@ -293,6 +295,25 @@ export class App extends PureComponent<Props, State> {
       document.body.classList.add("embedded")
     }
 
+    if (isInChildFrame()) {
```

Comment:
Can you add a comment to this block explaining what it generally accomplishes? (What does iframeResizer do for us, why do we need to assign it to `window.iFrameResizer`, and why do we only run this if we're in a child frame?)

Reply:
Good idea. I'll Put that in.